Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/jupyter_cache/executors/utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/nbclient/client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/nbclient/client.py", line 554, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/nbclient/client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/nbclient/client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from niworkflows.viz.notebook import display

display(
    fixed_path,
    moving_path,
    fixed_label="Predicted",
    moving_label="Left-out gradient",
);
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mTypeError[0m                                 Traceback (most recent call last)
[0;32m<ipython-input-4-4bcb8b7f79a7>[0m in [0;36m<module>[0;34m[0m
[1;32m      5[0m     [0mmoving_path[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m
[1;32m      6[0m     [0mfixed_label[0m[0;34m=[0m[0;34m"Predicted"[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 7[0;31m     [0mmoving_label[0m[0;34m=[0m[0;34m"Left-out gradient"[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      8[0m );

[0;32m/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/niworkflows/viz/notebook.py[0m in [0;36mdisplay[0;34m(fixed_image, moving_image, contour, cuts, fixed_label, moving_label)[0m
[1;32m     56[0m                     [0mlabel[0m[0;34m=[0m[0mmoving_label[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m
[1;32m     57[0m                     [0mcontour[0m[0;34m=[0m[0mcontour[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m
[0;32m---> 58[0;31m                     [0mcompress[0m[0;34m=[0m[0;32mFalse[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m     59[0m                 ),
[1;32m     60[0m             )

[0;32m/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/niworkflows/viz/utils.py[0m in [0;36mcompose_view[0;34m(bg_svgs, fg_svgs, ref, out_file)[0m
[1;32m    359[0m     [0;34m"""Compose the input svgs into one standalone svg with CSS flickering animation."""[0m[0;34m[0m[0;34m[0m[0m
[1;32m    360[0m     [0mout_file[0m [0;34m=[0m [0mPath[0m[0;34m([0m[0mout_file[0m[0;34m)[0m[0;34m.[0m[0mabsolute[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 361[0;31m     [0mout_file[0m[0;34m.[0m[0mwrite_text[0m[0;34m([0m[0;34m"\n"[0m[0;34m.[0m[0mjoin[0m[0;34m([0m[0m_compose_view[0m[0;34m([0m[0mbg_svgs[0m[0;34m,[0m [0mfg_svgs[0m[0;34m,[0m [0mref[0m[0;34m=[0m[0mref[0m[0;34m)[0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    362[0m     [0;32mreturn[0m [0mstr[0m[0;34m([0m[0mout_file[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m    363[0m [0;34m[0m[0m

[0;32m/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/niworkflows/viz/utils.py[0m in [0;36m_compose_view[0;34m(bg_svgs, fg_svgs, ref)[0m
[1;32m    394[0m     [0myoffset[0m [0;34m=[0m [0;36m0[0m[0;34m[0m[0;34m[0m[0m
[1;32m    395[0m     [0;32mfor[0m [0mi[0m[0;34m,[0m [0mr[0m [0;32min[0m [0menumerate[0m[0;34m([0m[0mroots[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 396[0;31m         [0mr[0m[0;34m.[0m[0mmoveto[0m[0;34m([0m[0;36m0[0m[0;34m,[0m [0myoffset[0m[0;34m,[0m [0mscale[0m[0;34m=[0m[0mscales[0m[0;34m[[0m[0mi[0m[0;34m][0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    397[0m         [0;32mif[0m [0mi[0m [0;34m==[0m [0;34m([0m[0mnsvgs[0m [0;34m-[0m [0;36m1[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    398[0m             [0myoffset[0m [0;34m=[0m [0;36m0[0m[0;34m[0m[0;34m[0m[0m

[0;31mTypeError[0m: moveto() got an unexpected keyword argument 'scale'
TypeError: moveto() got an unexpected keyword argument 'scale'

